### DISCUSSION TITLE
what is reclamation algorithm ?

### DISCUSSION THREAD

**Discussion by skzidan** [MaaAssistantArknights/MaaAssistantArknights#3827] [Q&A] [answered]
can anyone explain briefly?

**Comment by MistEO** [comment, marked as answer]
This is a new level of CN client, called "生息演算". maybe you can find its gameplay video on Bilibili.

**Comment by ABA2396** [comment]
It is a new activity of cn client. Although it is a limited time activity, it lasts for one month. We are not sure whether this activity will be permanent like I.S. , so we put the entrance here for the time being.
